xhr.bs

```diff
@@ -809,7 +809,7 @@ method must run these steps:
   <p>If the <a>synchronous flag</a> is unset, then:
 
   <ol>
-   <li><p><a>Fire a progress event</a> named <a event><code>loadstart</code></a> with 0 and 0.
+   <li><p><a>Fire a progress event</a> named <a event><code>loadstart</code></a> at [=this=] with 0 and 0.
```

Comment:
This seems like it should be the request object since a `loadstart` event is already being fired at the `XMLHttpRequestUpload` in the next step. However, since that's done conditionally and with different parameters, using `XMLHttpRequestUpload` wouldn't be completely redundant.

Reply:
This is correct.